_scripts/markdown/611.md

```diff
@@ -0,0 +1,13 @@
+## Fat Chicken
+
+A mutant chicken that can be found by chance when collecting an egg.
+
+This mutant reduces the wheat required to feed a chicken by 10%.
+
+There is a 1/1000 chance of producing a mutant chicken.
+
+## Supply
+
+Unlimited
+
+[Link](https://docs.sunflower-land.com/crafting-guide)
```

Comment:
I need to update the docs and point these to the new chicken info. Am doing this now.